benchmark/scripts/generate_harness/generate_harness.py

```diff
@@ -12,43 +12,70 @@
 #
 # ===---------------------------------------------------------------------===//
 
-# Generate CMakeLists.txt and utils/main.swift from templates.
+# Generate boilerplate, CMakeLists.txt and utils/main.swift from templates.
 
 from __future__ import print_function
 
-import glob
+import argparse
+import jinja2
```

Comment:
I’m not sure what you mean here: `jinja2` was used for templating the`CMakeLists.txt` and `utils/main.swift` before my changes. I’m trying to add `gyb` in order to re-use the skill I learned from other parts of swift `stdlib` to reduce boilerplate in tests. I’m not touching the `jinja2` part at all, outside of alphabetically ordering import declarations.